src/apis/FreeFire.js

```diff
@@ -0,0 +1,43 @@
+const { APIWrapper } = require('../')
+const fetch = require('node-fetch')
+
+const REQUEST_URL = 'https://ffstaticdata.switchblade.xyz'
+
+let weaponData = {}
+let languages = []
+
+module.exports = class FreeFire extends APIWrapper {
+  constructor () {
+    super({
+      name: 'freefire'
+    })
+  }
+
+  load () {
+    this.loadLanguages()
+    return this
+  }
+
+  async loadLanguages () {
+    const { locales } = await this.request('/metadata.json')
+    languages = locales
+  }
+
+  selectLanguage (language = 'en_US') {
+    const normalizedLanguage = language.slice(0, 2)
+    const matchingLanguage = languages.find(l => l.toLowerCase() === normalizedLanguage)
+
+    return matchingLanguage || 'en'
+  }
+
+  async getWeaponData (lang) {
+    const language = this.selectLanguage(lang)
+
+    if (!weaponData[language]) weaponData[language] = await this.request(`/${language}/weapons.json`)
```

Comment:
hm, i'm not sure about caching data from the api

Reply:
We already do that for the Legends of Runeterra command, where the API responds with a lot of information and requesting it again with every command run would slow stuff down too much. This command goes through the same thing.